Initial description of TSC

gdams opened the pull request.

Commit:
Initial description of TSC

karianna commented:
LGTM as a starting point but I expect a lot of people will want to discuss even this opening PR in detail.  Let's throw it to the #getopenjdk Slack channel and the adoption-discuss mailing list.

a-roberts reviewed (commented):
I propose

1) mandatory reelection process annually - to reduce complacency and encourage folks to want to be on the TSC
2) candidates for the committee should mention their ideas and intentions for the project

Draft wording would be 

The appointing of TSC members occurs on an annual basis and is based on a meritocracy. Candidates should be forthcoming with ideas for the project and welcoming to contributors (particularly new ones). Candidates with the intention of becoming on the TSC should briefly outline where they'd like to see the project going - all in a transparent manner that is available to the public.

tellison commented on README.md:
```diff
@@ -1,2 +1,50 @@
 # TSC
-The AdoptOpenJDK Technical Steering Comittee 
+The AdoptOpenJDK Technical Steering Comittee is the technical governing body of the AdoptOpenJDK organisation. It admits and oversees all top-level Projects in the AdoptOpenJDK organisation.
```
Based on the following list, are you equating "top level projects" with repositories in the org?
I guess we have to be aware of the difference / overlap / relationship between the Github org and the OpenJDK adoptopenjdk group.

tellison commented on README.md:
```diff
@@ -1,2 +1,50 @@
+* https://github.com/AdoptOpenJDK/openjdk-website-backend
+* https://github.com/AdoptOpenJDK/secrets _private_
+
+### List of TSC Responsibilities
```
While these responsibilities are good to see listed, I would hope that we are open to anyone coming to the project and contributing in these areas.  The goal should be to work in the open, and achieve consensus with all participants based on technical merit wherever possible.  I would not expect anyone to "flex their TSC credentials".

tellison commented on README.md:
```diff
@@ -1,2 +1,50 @@
+## TSC Members
+
+TSC members can nominate new members at any time. Candidates for membership tend to be people
+who have a competancy for community management and a high tolerance and patience for process
```
Can you explain "high tolerance and patience for process minutiae"?  I was expecting the TSC to be a *technical* merit body to guide the best interests of the org/projects.

tellison commented on README.md:
```diff
@@ -1,2 +1,50 @@
+|   |    |
+
+## Rules for TSC Membership
+* No single company can occupy more than 1/3 seats on the TSC
```
Is this necessary?  People should be working in the interests of the project, not their company.  If we see people pushing selfish corporate goals we'd need to address that promptly anyway.

What happens if the TSC becomes imbalanced, do we draw lots to kick people off?

gdams replied:
well when one member leaves we would simply re-elect but consider the balance

tellison commented on README.md:
```diff
@@ -1,2 +1,50 @@
+
+## Rules for TSC Membership
+* No single company can occupy more than 1/3 seats on the TSC
+* The appointing of TSC members occurs on an annual basis and is based on a meritocracy. 
```
Is the full card appointed each year, or people serve a one-year term?  Would be good to have some continuity of experience.

tellison reviewed (commented):
Thanks for starting this description, it raises a number of good questions.

I've added a few thoughts as separate comments to the doc.

karianna committed:
Updated Readme to start the knowledge base

Updated Readme with Shelly, Stuart and George so we can start the knowledge base

karianna reviewed (approved):
Good for a starting point to discuss - we also want to build out the knowledge base from here, hence merging.
